tracer/src/Datadog.Trace/RemoteConfigurationManagement/RcmCapabilitiesIndices.cs

```diff
@@ -85,6 +85,14 @@ internal static class RcmCapabilitiesIndices
 
         public static readonly BigInteger AsmRaspCmd = Create(37);
 
+        public static readonly BigInteger EnableDynamicInstrumentation = Create(38);
+
+        public static readonly BigInteger EnableExceptionReplay = Create(39);
+
+        public static readonly BigInteger EnableCodeOrigin = Create(40);
+
+        public static readonly BigInteger EnableLiveDebugging = Create(41);
+
```

Comment:
Did we figure out the source of truth for this? I remember there being a conflict here 😅

(I'm guessing "yes" since you're posting this PR, just wanted to confirm)

Reply:
😅 yes the source of truth has changed from confluence to the code
https://github.com/DataDog/dd-go/pull/169590